sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/Utils.java

```diff
@@ -16,16 +19,16 @@
 
 public final class Utils {
 
-    private static final Gson sGson = new Gson();
+    private static final Gson S_GSON = new Gson();
```

Comment:
remove S_ for constants, sField is the static field for Android source convention when not a constant.

Also static final does not mean compile time constant, here sField works as it's not compile time constant expression (as opposed to string or int).

Reply:
Basically if it cannot be used as annotation parameter, its not constant compile time.